cmd/juju/application/deploy_test.go

```diff
@@ -406,6 +409,30 @@ func (s *DeploySuite) TestResources(c *gc.C) {
 	})
 }
 
+func (s *DeploySuite) TestLXDProfileLocalCharm(c *gc.C) {
```

Comment:
tests for non local charms?

Reply:
https://trello.com/c/KUkeXyJT/592-update-charm-store-repo-deploy-to-check-for-the-presence-of-the-profile-file-and-validate-the-contents